resources/app/src/components/people/DemographicsForm.vue

```diff
@@ -933,7 +945,44 @@ export default {
             if (this.isFormValid) {
                 alert('Form is valid and ready to be submitted!');
                 items = {
-                    ...this.formdata,
+                  //  ...this.formdata,
```

Comment:
What problem did it cause to use the '...' assignment? This runs the risk of leaving something out.